src/Material/Button.elm

```diff
@@ -145,6 +232,26 @@ ripple =
     (\options -> { options | ripple = True })
 
 
+{-| Add link-attributes that turn the button from `button` into a `a`-element.
+This allows for links that look and feel like buttons but perform link actions.
+
+    Button.render ...
+      [ ...
+      , Button.link
+          [ Button.href "#some-link"
+          , Button.target "_blank"
+          ]
+      ]
+      [ ... ]
+
+**NOTE** An empty list keeps the element as `button`
```

Comment:
I suggest making this not true by switching to `linkOptions : Maybe (List ...)`. I know its a corner-case that very few people will care about, but I think its better that the API contains as few surprises as possible.